### PR TITLE
Fix #125

### LINKED ISSUE #125: Fix failing Devanagari क tests
Currently we skip these tests from the Unicode test suite (see #124):
https://github.com/unicode-rs/unicode-segmentation/blob/6191f8e287a5cd534af54974952957a6e7bdf6a1/src/test.rs#L53

We should fix whatever issue is causing these to fail.

### PR DESCRIPTION
**Pull request by Jules-Bertholet**
Fixes #125

### CODE REVIEW

File: scripts/unicode.py [hunk 1]
```diff
@@ -431,3 +431,22 @@ def emit_break_module(f, break_table, break_cats, name):
             sentence_table.extend([(x, y, cat) for (x, y) in sentence_cats[cat]])
         sentence_table.sort(key=lambda w: w[0])
         emit_break_module(rf, sentence_table, list(sentence_cats.keys()), "sentence")
+
+        rf.write("""
```
Comment (Manishearth) [inline scripts/unicode.py:457]: nit: no, let's not have too much code generation like this. they can live in a different module somewhere

**Review by Manishearth — CHANGES_REQUESTED** [review]

File: .github/workflows/rust.yml [hunk 2]
```diff
@@ -32,6 +32,6 @@ jobs:
     - name: Rustfmt
       run: cargo fmt --all --check
     - name: Verify regenerated files
-      run: ./scripts/unicode.py && diff tables.rs src/tables.rs
+      run: ./scripts/unicode.py && rustfmt tables.rs && diff tables.rs src/tables.rs
```
Comment (Manishearth) [inline .github/workflows/rust.yml:35]: issue: no, let's just `#[rustfmt::skip]` the module so people don't have to remember to do this.

Comment (Jules-Bertholet) [reply .github/workflows/rust.yml:35]: Should we also start `skip`ping testdata.rs?

Comment (Manishearth) [reply .github/workflows/rust.yml:35]: sure yeah

File: src/grapheme.rs [hunk 3]
```diff
@@ -452,6 +462,38 @@ impl GraphemeCursor {
         }
     }
 
+    #[inline]
+    fn handle_incb_consonant(&mut self, chunk: &str, chunk_start: usize) {
```
Comment (Manishearth) [inline src/grapheme.rs:466]: issue: please add comments explaining what this code does (doc comments) and an inline ones explaiining how it's trying to achieve that

**Review by Manishearth — APPROVED** [review]

**Pull request merged**